packages/block-editor/src/components/preview-options/index.js

```diff
@@ -45,6 +46,12 @@ export default function PreviewOptions( {
 		desktop,
 	};
 
+	const devices = [
```

Comment:
See what term fits best here; there's `deviceType` in many places, and here I shortened it to `device.